test/core.js

```diff
@@ -3,625 +3,601 @@
  */
 
 import assert from 'node:assert/strict'
-import {promises as fs, renameSync} from 'node:fs'
+import {promises as fs, renameSync, existsSync} from 'node:fs'
 import process from 'node:process'
 import {URL, pathToFileURL} from 'node:url'
 import test from 'node:test'
 import semver from 'semver'
 import {resolve} from '../index.js'
 
 const windows = process.platform === 'win32'
-const veryOldNode = semver.lt(process.versions.node, '16.0.0')
 const oldNode = semver.lt(process.versions.node, '18.0.0')
 
+const run = (/** @type {() => void} */ f) => f()
+
 process.on('exit', async () => {
   // Has to be sync.
-  renameSync('package.json.bak', 'package.json')
+  if (existsSync('package.json.bak')) {
+    renameSync('package.json.bak', 'package.json')
+  }
 })
 
-test('resolve(specifier, base?, conditions?)', async function () {
-  assert(resolve, 'expected `resolve` to exist (needed for TS in baseline)')
+test(
+  'resolve(specifier, base?, conditions?)',
+  {skip: false},
```

Comment:
Is this line a left over from some local testing?

Reply:
Ah, no. Maybe I should put in a comment about this. 

Look at `script.js`, which modifies `baseline.js` to change `fskip: alse` to "skip if version < 20", i.e. skip checking the sync version of `import.meta.resolve` if v<20.

I tried just skipping the whole test (using an `if (...) process.exit(0)` at the beginning of the file), but the coverage raised an alarm about not enough coverage. Only skipping the test worked. Not sure why...

The alternative for replacing the `skip` would be for `script.js` to generate the sync baseline test only if v > 20, but I didn't like that for some reason. Can't exactly point the finger to why.

Reply:
Oh smart Hmm, sad about the coverage lines.

A comment is a good idea! Something like this?
```
// Note: this is toggled by the reference tests.
```

Reply:
Added comment!